### DISCUSSION TITLE
[HELP] get error "TypeError: Cannot read properties of undefined (reading 'replace')"

### DISCUSSION THREAD

**Discussion by hapnan** [MasterKale/SimpleWebAuthn#573] [Troubleshooting]
![image](https://github.com/MasterKale/SimpleWebAuthn/assets/34102619/0cc16af0-012f-4d71-8af1-e1ed05845967)
hi, I get an error when I want to do startRegistration(), even though all the data has been placed correctly

I put the code below
```
const resp = await fetch(
      "http://localhost:3000/api/registration?nim=${data.nim}",
      {
        method: "GET",
        headers: {
          "Content-Type": "application/json",
        },
      },
    );

    let attResp: RegistrationResponseJSON;
    try {
      attResp = await startRegistration(await resp.json());
      console.log("Registration Response :", JSON.stringify(attResp));
    } catch (error) {
      if (error instanceof Error) {
        if (error.name == "InvalidStateError") {
          console.log(
            "Error: Authenticator was probably already registered by user",
          );
        } else {
          console.log(error);
        }
      }
      throw error;
    }
```

**Comment by wparad** [comment]
What is the value of `await resp.json()` here?

**Reply by hapnan** [reply]
response of generateRegistrationOptions()

**Comment by kingsley-4123** [comment]
Please what's the solution to this error cus am getting the same error

**Reply by MasterKale** [reply]
How are you calling startRegistration? Please include  all of the values you're passing in to the method such that I can see every value of every option you're specifying.
